pkg/injector/pod_patch.go

```diff
@@ -114,7 +114,7 @@ func (i *injector) getPodPatchOperations(ar *v1.AdmissionReview,
 	}
 
 	// Keep DNS resolution outside of getSidecarContainer for unit testing.
-	placementAddress := fmt.Sprintf("%s:80", getKubernetesDNS(placementService, namespace))
+	placementAddress := fmt.Sprintf("%s:50005", getKubernetesDNS(placementService, namespace))
```

Comment:
will this resolve to multiple addresses?

Reply:
right. please read kubernetes headless service. we use this way for service invocation.